validator/sawtooth_validator/journal/block_validator.py

```diff
@@ -408,6 +408,8 @@ def process_block_verification(self, block, callback):
         except Exception:  # pylint: disable=broad-except
             LOGGER.exception(
                 "Block validation failed with unexpected error: %s", block)
+        else:
+            callback(block)
```

Comment:
The chain controller still needs to send a message to the consensus engine about the blocks invalidity

Reply:
Nevermind, I think I misread how this was behaving (there's been a sudden increase of the use of `else` in `except` clauses recently)